contracts/modules/TransferManager/BlacklistTransferManager.sol

```diff
@@ -0,0 +1,244 @@
+pragma solidity ^0.4.24;
+
+import "./ITransferManager.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+ * @title Transfer Manager module for manually approving or blocking transactions between accounts
+ */
+contract BlacklistTransferManager is ITransferManager {
+    using SafeMath for uint256;
+
+    bytes32 public constant ADMIN = "ADMIN";
+    
+    struct BlacklistsDetails {
```

Comment:
Can we use second consistently and rename to `startTime`, `endTime`, `repeatPeriodTime`.